### ISSUE TITLE
Unclear output for certonly option: Where is the certificate

### ISSUE THREAD

**Issue by maniac0s** [certbot/certbot#6848]
## My operating system is (include version):

Debian 9

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):

git clone github...

## I ran this command and it produced this output:

	root@test ~/certbot # ./certbot-auto -d *.somedomain.net --manual --preferred-challenges dns-01 certonly
	Saving debug log to /var/log/letsencrypt/letsencrypt.log
	Plugins selected: Authenticator manual, Installer None
	Obtaining a new certificate
	Performing the following challenges:
	dns-01 challenge for somedomain.net

	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
	NOTE: The IP of this machine will be publicly logged as having requested this
	certificate. If you're running certbot in manual mode on a machine that is not
	your server, please ensure you're okay with that.

	Are you OK with your IP being logged?
	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
	(Y)es/(N)o: y

	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
	Please deploy a DNS TXT record under the name
	_acme-challenge.somedomain.net with the following value:

	***

	Before continuing, verify the record is deployed.
	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
	Press Enter to Continue
	Waiting for verification...
	Cleaning up challenges
	archive directory exists for somedomain.net

## Certbot's behavior differed from what I expected because:

I don't know what happened. I get no clear information on what it did. I don't know where the certificates are saved. I don't know if the process was successful (I guess? There was no error...)

Certificates in archive directory are untouched for months.

Edit: It seems there was an error but I got no error output, only the log reveals:

	019-03-12 11:07:19,129:DEBUG:certbot.log:Exiting abnormally:
	Traceback (most recent call last):
	  File "/opt/eff.org/certbot/venv/bin/letsencrypt", line 11, in <module>
		sys.exit(main())
	  File "/opt/eff.org/certbot/venv/local/lib/python2.7/site-packages/certbot/main.py", line 1365, in main
		return config.func(config, plugins)
	  File "/opt/eff.org/certbot/venv/local/lib/python2.7/site-packages/certbot/main.py", line 1250, in certonly
		lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
	  File "/opt/eff.org/certbot/venv/local/lib/python2.7/site-packages/certbot/main.py", line 121, in _get_and_save_cert
		lineage = le_client.obtain_and_enroll_certificate(domains, certname)
	  File "/opt/eff.org/certbot/venv/local/lib/python2.7/site-packages/certbot/client.py", line 428, in obtain_and_enroll_certificate
		self.config)
	  File "/opt/eff.org/certbot/venv/local/lib/python2.7/site-packages/certbot/storage.py", line 1004, in new_lineage
		"archive directory exists for " + lineagename)
	CertStorageError: archive directory exists for somedomain.net

Edit2: Just running exact command again gives a new certificate and no crash in debug log anymore. This is very confusing behaviour.

	root@test ~/certbot # ./certbot-auto -d *.somedomain.net --manual --preferred-challenges dns-01 certonly
	Saving debug log to /var/log/letsencrypt/letsencrypt.log
	Plugins selected: Authenticator manual, Installer None
	Obtaining a new certificate

	IMPORTANT NOTES:
	 - Congratulations! Your certificate and chain have been saved at:
	   /etc/letsencrypt/live/somedomain.net-0001/fullchain.pem
	   Your key file has been saved at:
	   /etc/letsencrypt/live/somedomain.net-0001/privkey.pem
	   Your cert will expire on 2019-06-10. To obtain a new or tweaked
	   version of this certificate in the future, simply run certbot-auto
	   again. To non-interactively renew *all* of your certificates, run
	   "certbot-auto renew"
	 - If you like Certbot, please consider supporting our work by:

	   Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
	   Donating to EFF:                    https://eff.org/donate-le

**Comment by adferrand** [comment]
Ah. I was about to ask you to give me some listings of your `/etc/letsencrypt` directory. I think that for some reasons, the layout of your `/etc/letsencrypt` was broken. But the first execution of certbot reconstruct the correct layout, and the second execution had taken advantage of it.

I could not reproduce on my own instances, so I think the problem was specific to some of your system particularities.

Could you confirm that another arbitrary domain is working ? If so, can we consider the issue as solved ?

**Comment by maniac0s** [comment]
I don't know how to check that without having more domains. I am also quite involved in other tasks right now. I am just wondering why it didn't throw an error at the first run, I searched for the certificates for a while until I saw the note that it wrote to a debug log. The output of

    Press Enter to Continue
    Waiting for verification...
    Cleaning up challenges
    archive directory exists for somedomain.net

didn't indicate any problem. Also I didn't change the directories in /etc/letsencrypt/ since the last run of certbot.
